sysutils/tmux/Portfile

```diff
@@ -32,6 +32,9 @@ license         BSD
 
 depends_lib     port:libevent port:ncurses
 
+# Use screen by default to display tmux correctly when backspacing
+configure.args-append   --with-TERM=screen
```

Comment:
Since you're changing a configure argument here and the build will differ from the current situation; you should increase the `revision` so that everyone gets the new build. That likely applies to both the `tmux` and `tmux-devel` ports.